content/applications/hr/payroll.rst

```diff
@@ -152,7 +301,7 @@ Time off options section
 Reporting section
 *****************
 
-- :guilabel:`Unforeseen Absence`: If the work entry should be visible on the unforeseen absences
+- :guilabel:`Unforeseen Absence`: if the work entry should be visible on the unforeseen absences
   report, check this box.
 
 .. image:: payroll/new-work-entry.png
```

Comment:
It's unclear to me why the title is boxed in red on the screenshot.

Also a general comment about images: it appears the compression is pretty aggressive for most of these screenshots in the doc. Some are barely legible, can you take a look?

Reply:
I agree- I retook a bunch of these images. I ended up zooming in more and cropping some things that were blank/not filled in so I could get a smaller image/field. The one that I just can't get much better is this one, since it's almost the entire screen, and I was trying to get all the info in the shot.

The smaller the section I need, the better I can make it look since I can zoom in nice. But when it's zoomed out and I need a big chunk. it always looks super blurry and I can't get it crisper. If you look at the others, I was able to zoom in more and crop more, which ended up with a sharper image

For images like the working schedules- I cropped it at Wednesday, which makes for a much sharper image. Even though it doesn't show the whole week, I felt it showed enough to convey the idea. I thought about having a note saying 'the image does not display the full schedule for the whole week' but I didn't think that was a good precedent.